src/services/handler/acknowledgement-handler.js

```diff
@@ -0,0 +1,24 @@
+import { parseConversationId } from "../parser/parsing-utilities";
+import { logError, logInfo } from "../../middleware/logging";
+import { setCurrentSpanAttributes } from "../../config/tracing";
+import { ACKNOWLEDGEMENT_TYPES } from "../../constants/acknowledgement-types";
+import { parseAcknowledgementMessage } from "../parser/acknowledgement-parser";
+
+export const acknowledgementMessageHandler = async message => {
+  const conversationId = await parseConversationId(message);
+  const { acknowledgementTypeCode } = await parseAcknowledgementMessage(message);
+
+  setCurrentSpanAttributes({ conversationId });
+
+  switch (acknowledgementTypeCode) {
+    case ACKNOWLEDGEMENT_TYPES.POSITIVE.includes(acknowledgementTypeCode):
+      logInfo(`POSITIVE ACKNOWLEDGEMENT RECEIVED`);
+      break;
+    case ACKNOWLEDGEMENT_TYPES.NEGATIVE.includes(acknowledgementTypeCode):
+      logInfo(`NEGATIVE ACKNOWLEDGEMENT RECEIVED`);
+      break;
+    default:
+      logError(`ACKNOWLEDGEMENT TYPE ${acknowledgementTypeCode} IS UNKNOWN.`);
+      break;
+  }
+};
```

Comment:
Can you check whether this switch case block really work as expected?

I tried to test this bit of code with several values:
When acknowledgementTypeCode is "AA", it went to default block
When acknowledgementTypeCode is "AE", it went to default block
When acknowledgementTypeCode is boolean false, it logged (`POSITIVE ACKNOWLEDGEMENT RECEIVED`)


The issue seems to be that, switch case compare the value `acknowledgementTypeCode` with `ACKNOWLEDGEMENT_TYPES.POSITIVE.includes(acknowledgementTypeCode)`, the prev one is "AA" but latter one is `true`, which are not the same, and then it didn't went into the branch